x-pack/plugins/ml/public/jobs/new_job_new/common/job_creator/configs/datafeed.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { IndexPatternTitle } from '../../../../../../common/types/kibana';
+import { JobId } from './job';
+export type DatafeedId = string;
+
+export interface Datafeed {
```

Comment:
Suggest to move this to the overall `common/types/` directory. We already have `jobs.ts` there, which at the moment only has `datafeed_config: object;` right now but then could also reference this interface.